notebooks/curation_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Discover API notebooks
+
+The Discover API notebooks are pre-baked scripts to facilitate use of the 
+[Discover API](https://api.cellxgene.cziscience.com/curation/ui/#/), which itself supports 
+curation flow interactions with the [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) tool.
+
+## Notebook usage
+
+Many notebooks have a few fields that require user input, such as the relevant Collection id or Dataset id.
+
+### Python notebooks: `python/`
+
+These notebooks are the most user-friendly; start here if you're unsure. All access token generation, url construction, and response 
+handling is abstracted away from the user. Unitary interactions such as getting a Dataset, creating a Collection, etc., 
```

Comment:
_creating a collection_ is a curator action, not a consumer action.

Reply:
Correct. Do you mean to say that, as such, it should not be used as an example? Only consumer actions should be referenced here?